### DISCUSSION TITLE
Drifters

### DISCUSSION THREAD

**Discussion by ciarapimm** [ECCO-Hackweek/ecco-2024#154] [General]
We will chat about the project here

**Comment by gaelforget** [comment]
tentative list of next steps :

- [x] bring team members up to speed with Julia
- [x] add environment (toml) + initial_simulations.jl to project repo
- [x] run initial simulations with IndividualDisplacements & OCCA
- [ ] zoom in on South Atlantic + Indian region, define mask (see https://github.com/ECCO-Hackweek/EH24-Drifters/issues/3)
- [ ] setup particle release at Northern Section  (see https://github.com/ECCO-Hackweek/EH24-Drifters/issues/3)
- [ ] detect exit through the other sides of mask
- [ ] calculate transit time distributions
- [ ] attach volume transport to particles (entry) and quantify exit points
- [ ] test sensitivity of results to number of particles 
- [ ] calculate stream function and compare to Eulerian calculation

see below for suggestions on how to do some of these

**Comment by gaelforget** [comment]
See https://github.com/ECCO-Hackweek/EH24-Drifters/blob/main/scripts/README.md

**Comment by gaelforget** [comment]
For `detect exit through the other sides of mask` ...

**Comment by ciarapimm** [comment]
[drifters.pptx](https://github.com/user-attachments/files/17440199/drifters.pptx)
